src/app/components/annotations/pages/verification/verification.component.ts

```diff
@@ -246,18 +234,107 @@ class VerificationComponent
     this.hasUnsavedChanges = false;
   }
 
+  protected handleDecision(decisionEvent: Event): void {
+    if (!this.isDecisionEvent(decisionEvent)) {
+      console.error("Received invalid decision event", decisionEvent);
+      return;
+    }
+
+    this.hasUnsavedChanges = true;
+
+    const decision = decisionEvent.detail;
+    for (const [subject, receipt] of decision) {
+      const change = receipt.change;
+
+      if (Object.prototype.hasOwnProperty.call(change, "verification")) {
+        this.handleVerificationDecision(subject);
+      }
+
+      if (Object.prototype.hasOwnProperty.call(change, "newTag")) {
+        this.handleTagCorrectionDecision(subject);
+      }
+    }
+  }
```

Comment:
TODO: we should be updating our annotation model after these updates.

In particular it should now have a new tagging and a set of verifications. This can wait until we start showing verifications in history